lib/ts/unit-tests/test_string_view.cc

```diff
@@ -310,7 +310,11 @@ TEST_CASE("Capacity", "[string_view] [capacity]")
     REQUIRE(sv.size() == 0);
     REQUIRE(sv.length() == 0);
     REQUIRE(sv.empty() == true);
+#if __cplusplus >= 201703L
```

Comment:
I think @SolidWallOfCode case for this one is better as it is possible different impls will report different values depending on how the treat max_size()